### PR TITLE
aa-modules-upgrade-bug-RN.md

### PR DESCRIPTION
**Pull request by AlonMagrafta**

### CODE REVIEW

File: content/rs/release-notes/rs-6-2-18.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ This version of Redis Enterprise Software offers:
 - Compatibility with open source Redis v6.2.6
 - Additional enhancements and fixes
 
+### Known upgrade limitation
+Do not upgrade a cluster that hosts Active-Active databases with module/s before following the next steps:
```
Comment (nermiller) [inline content/rs/release-notes/rs-6-2-18.md]:
```suggestion
Before you upgrade a cluster that hosts Active-Active databases with modules, ensure you complete these steps:
```

File: content/rs/release-notes/rs-6-2-18.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ This version of Redis Enterprise Software offers:
...
+Check the module and its version as it appears in the db config and in the default db config using the `crdb-cli crdb get --crdb-guid <crdb-guid>`. The fields are called 'modules' and 'module_list'.
```
Comment (nermiller) [inline content/rs/release-notes/rs-6-2-18.md]:
```suggestion
1. Use the `crdb-cli crdb get --crdb-guid <crdb-guid>` command to check that the modules (`modules`) and their versions (`module_list`) are as they appear in the db config and in the default db config. 
```

File: content/rs/release-notes/rs-6-2-18.md [hunk 3]
```diff
@@ -24,6 +24,12 @@ This version of Redis Enterprise Software offers:
...
+Via the UI Admin console, under the 'redis modules' tab, validate that these module/s with their specific version/s are loaded to the cluster.
```
Comment (nermiller) [inline content/rs/release-notes/rs-6-2-18.md]:
```suggestion
1. From the admin console's **Redis Modules** tab, validate that these modules with their specific versions are loaded to the cluster.
```

File: content/rs/release-notes/rs-6-2-18.md [hunk 4]
```diff
@@ -24,6 +24,12 @@ This version of Redis Enterprise Software offers:
...
+In case one or more of the modules+versions are missing, or if you need help [contact Redis support](https://redis.com/company/support/) before taking additional steps.
```
Comment (nermiller) [inline content/rs/release-notes/rs-6-2-18.md]:
```suggestion
If one or more of the modules/versions are missing, or if you need help, please [contact Redis support](https://redis.com/company/support/) before taking additional steps.
```

File: content/rs/release-notes/rs-6-4-2.md [hunk 5]
```diff
@@ -192,6 +192,13 @@ See [Upgrade modules](https://docs.redis.com/latest/modules/install/upgrade-modu
 
 - Added an alert to notify when a node operation (such as maintenance mode) failed, aborted, or was canceled. The alert is enabled by default (RS76089)
 
+#### Known upgrade limitation
+Do not upgrade a cluster that hosts Active-Active databases with module/s before following the next steps:
```
Comment (nermiller) [inline content/rs/release-notes/rs-6-4-2.md]: See my suggestions above.

**Review by nermiller — COMMENTED** [review]
A few edits. Rachel, please keep me honest about the UI naming and fields

**Review by rrelledge — APPROVED** [review]

**Pull request merged**